README.md

```diff
@@ -1 +1,46 @@
-# Haal-Centraal-BRP-historie-bevragen
\ No newline at end of file
+# Haal-Centraal-BRP-historie-bevragen
+
+![lint oas](https://github.com/VNG-Realisatie/Haal-Centraal-BRP-historie-bevragen/workflows/lint-oas/badge.svg)
+![generate sdks](https://github.com/VNG-Realisatie/Haal-Centraal-BRP-historie-bevragen/workflows/generate-sdks/badge.svg)
+![generate postman collection](https://github.com/VNG-Realisatie/Haal-Centraal-BRP-historie-bevragen/workflows/generate-postman-collection/badge.svg)
+
+BRP Historie Bevragen is een Haal Centraal API voor het zoeken en raadplegen van historiesche gegevens met betrekking op een ingeschreven natuurlijke personen voor alle binnengemeentelijke afnemers in NL. De informatie die de API levert is herleidbaar naar het LO GBA 3.10.
```

Comment:
["Het Logisch Ontwerp versie 3.12 is op 1 juli 2019 in werking getreden."](https://www.rvig.nl/documenten/publicaties/2019/06/26/logisch-ontwerp-gba-versie-3.12)

Dus ik zou hier niet 3.10 noemen. Dat is wel erg oud.

Reply:
Aangepast